odoo_updates/odoo_updates.py

```diff
@@ -266,6 +350,53 @@ def diff_to_screen(views_states, title):
                     click.secho(line)
 
 
+def fields_to_screen(fields_states, title):
+    show_model_field = dict()
+    for state, values in fields_states.iteritems():
+        click.secho('+ {state} {title}'.
+                    format(state=state.title(), title=title), fg='yellow')
+        for field in values:
+            if state == 'updated':
+                column = {'type': difflib.unified_diff(
+                        field['original'].get('type', '').split('\n'),
+                        field['modified'].get('type', '').split('\n')),
+                        'field_description': difflib.unified_diff(
+                            field['original'].get('field_description', '').split('\n'),
+                            field['modified'].get('field_description', '').split('\n')), }
+                field.update(column)
+            else:
+                field.update({'type': field['type'].split('\n'),
+                              'field_description': field['field_description'].split('\n'), })
+            if field.get('model') not in show_model_field:
+                show_model_field[field.get('model')] = []
+                click.secho('+++ {title} {model}'.
+                            format(title='model',
+                                   model=field.get('model')),
+                            fg='yellow')
+            if field.get('name') not in\
+               show_model_field[field.get('model')]:
+                show_model_field[field.get('model')].append(field.get('name'))
+                click.secho('+++ {title} {name}'.
+                            format(title='field', name=field.get('name')),
+                            fg='yellow')
+            for colm in field:
+                if 'field' in colm:
+                    output = colm.replace("_", " ")
+                else:
+                    output = "field {out}".format(out=colm)
+                diff = list(field.get(colm))
+                if diff and colm in ('type', 'field_description'):
+                    click.secho('++++ {column}'.format(column=output),
+                                fg='yellow')
```

Comment:
@mapuerta  Same thing here, you are already displaying the model and the name of the field, just display the type and description like this:
click.secho('+++ type {type}'.format(type=field['type']), fg='yellow')
click.secho('+++ field description {description}'.format(description=field['field_description']), fg='yellow')
instead of using those if-else statements and the for cycle